src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+
+    private static final int MAX_VALUE = 5;
+    private static final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public static Figure getDefaultFigure() {
+        return new Circle("White", 10);
+    }
```

Comment:
The 'getDefaultFigure()' method should not be static according to the checklist. Static methods are generally discouraged in this context. Instead, create an instance of the class to call this method.